KsApi/models/lenses/CountryLenses.swift

```diff
@@ -0,0 +1,29 @@
+import Prelude
+
+extension Project.Country {
+  public enum lens {
```

Comment:
I think this is fine, we should probably figure out which approach we want to use going forward considering that we changed our other models to just be `public var` properties in order to use keypath lenses 🤔